packages/fela/src/bindings/connectFactory.js

```diff
@@ -59,7 +60,15 @@ export default function connectFactory(
         }
       }
 
-      return EnhancedComponent
+      // Re-attach static methods
+      EnhancedComponent.childContextTypes = component.childContextTypes;
```

Comment:
There're only a few React-specific ones right? Doesn't seem like to much overhead for the other projects right now. We could refactor that later.